changelog.config.js

```diff
@@ -0,0 +1,82 @@
+module.exports = {
```

Comment:
I would suggest to install the cz adapter (as a npm devdep) and reference it in cz conf.

Reply:
can you still have granular control like we have here? git-cz is the adapter i'm using

Reply:
I think we should use the default config from the adapter without having it in our workspace, then maybe override it when we'll need to.